packages/api/src/ontology/ActionDefinition.ts

```diff
@@ -20,40 +20,67 @@ import type {
   ReleaseStatus,
 } from "./ObjectTypeDefinition.js";
 
-export interface ActionMetadata<
-  A extends string,
-  K extends string,
-> {
+export interface ActionMetadata {
   type: "action";
-  apiName: A;
+  apiName: string;
   description?: string;
   displayName?: string;
-  parameters: Record<any, ActionParameterDefinition<K, any>>;
-  modifiedEntities?: Partial<Record<K, ActionModifiedEntity>>;
+  parameters: Record<any, ActionMetadata.Parameter<any>>;
```

Comment:
Do we never need to specify this?

Reply:
If you mean the K, it seems to work just fine without it.